### ISSUE TITLE
no jep in java.library.path (java.lang.UnsatisfiedLinkError) on examples

### ISSUE THREAD

**Issue by antonkulaga** [xtreamsrl/polynote-docker#2]
Whenever I try to run examples with latest docker container I get:
```
no jep in java.library.path (java.lang.UnsatisfiedLinkError)
```

**Comment by antonkulaga** [comment]
It may have something to do with https://github.com/polynote/polynote/issues/700

**Comment by rmaganza** [comment]
Thank you for your input!
This was finally fixed by 0cad0891d6922dc7fc822f00b56f7e840d0158ed.
